components/MyCollectionList/index.js

```diff
@@ -0,0 +1,38 @@
+import useSWR from "swr";
+import Flex from "@/components/Layout/Flex";
+import StarshipCollectionCard from "../StarshipCollectionCard";
+
+export default function MyCollectionList() {
+  const { data: users, isLoading } = useSWR("/api/users");
+  if (!users || isLoading) {
+    return <h1>Loading...</h1>;
+  }
+
+  const mainUser = users[0];
+
+  const starshipsForSale = mainUser.listings.map((listing) => listing.starship);
+
+  const starshipsNotForSale = mainUser.starships.filter((starship) => {
+    if (!starshipsForSale.includes(starship._id)) {
+      return starship;
+    }
+    return;
+  });
+
+  return (
+    <>
+      <Flex gap="1.5rem" flexWrap="wrap" alignItems="center" direction="column">
+        {starshipsNotForSale.map((starship) => {
+          return (
+            <StarshipCollectionCard
+              {...starship}
+              key={starship._id}
+              name={starship.name}
+              img={starship.img}
+            />
+          );
+        })}
+      </Flex>
+    </>
```

Comment:
Mapping looks good, we would suggest some basic html structure. A `<main>` tag around the content. An `<ul>` around the mapping progress, and `<li>`later in the StarshipCollectionCard around the single items.
```suggestion
    <main>
    <ul>
      <Flex gap="1.5rem" flexWrap="wrap" alignItems="center" direction="column">
        {starshipsNotForSale.map((starship) => {
          return (
            <StarshipCollectionCard
              {...starship}
              key={starship._id}
              name={starship.name}
              img={starship.img}
            />
          );
        })}
      </Flex>
      </ul>
    </main>
```

Reply:
Thank you for the review! We do not want to use the UL element, but we will replace the cards as a section element, instead of a div, so our HTML structure is semantic.  We will add the main element as well. @DaveKrauss